src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -754,6 +756,88 @@ private async Task<string> ValidateFreeSpace(RestoreBackupConfigurationBase rest
             return restoreBackup.DatabaseName;
         }
 
+        [RavenAction("/admin/backup/database", "POST", AuthorizationStatus.DatabaseAdmin)]
+        public async Task BackupDatabaseOnce()
+        {
+            PeriodicBackupRunner.CheckServerHealthBeforeBackup(ServerStore, "BackupDatabaseOnce");
+
+            using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            {
+                var json = await context.ReadForMemoryAsync(RequestBodyStream(), "database-backup");
+                var backupConfiguration = JsonDeserializationServer.OneTimeBackupConfiguration(json);
+                ServerStore.ConcurrentBackupsCounter.StartBackup(backupConfiguration.Name, Logger);
+                ServerStore.LicenseManager.AssertCanAddPeriodicBackup(backupConfiguration);
```

Comment:
What if this fails?
Need to run `ConcurrentBackupsCounter.FinishBackup` or better move it just before starting the operation